### ISSUE TITLE
model.predict outputting logits not probs

### ISSUE THREAD

**Issue by mhking71** [AIH-SGML/MixMIL#20]
I’ve followed pretty closely with the binary classification example for histology, but it seems like when I use the model.predict method l it outputs logits not probs (I have negative values). In the example, you don’t exp them or anything to put them back into probability space. I’ve like triple checked to make sure my code is exactly like yours and I’m pretty sure it is, though there could still be a bug.

Any advice?

Best,
Mike

**Comment by jan-engelmann** [comment]
Hi Mike,
Thanks for reaching out!
I assume you're talking about binomial likelihood with n_trials=1?
In the example we are looking at logits since we use AUC to judge predictive performance which doesn't need probabilities. 
You can use the sigmoid function for the above case or softmax for categorical likelihood with more than 2 classes. 
Cheers,
Jan
